tox.ini

```diff
@@ -76,6 +76,7 @@ deps =
     {[testenv]deps}
     behave==1.2.4
     behave-pytest==0.1.1
+    /root/mesos.native-0.28.2-py27-none-any.whl
```

Comment:
I also wonder whether this is going to cause us to have a specific version dependency on the system mesos package...